transactions/WFLY-14869-microprofile-lra.adoc

```diff
@@ -0,0 +1,105 @@
+= Support for MicroProfile LRA
+:author:            Ondrej Chaloupka
+:email:             ochaloup@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:keywords:          transaction, saga, lra, microprofile
+
+== Overview
+
+The goal of this RFE is providing the Galleon feature pack for WildFly
+which integrates the https://github.com/jbosstm/narayana/tree/5.11.2.Final/rts/lra[Narayana implementation]
+of the https://github.com/eclipse/microprofile-lra[MicroProfile LRA specification].
+
+The MicroProfile LRA is a transaction processing pattern well suited for microservice development
+based on the saga pattern. The motivation of such processing is summarized in the https://github.com/eclipse/microprofile-lra/blob/1.0/README.adoc#motivation[MicroProfile Spec readme].
+
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-14869
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/EAP7-1496
+
+=== Dev Contacts
+
+* mailto:{ochaloup@redhat.com}[{author}]
+* TBD
+
+=== QE Contacts
+
+* TBD
+
+=== Testing By
+
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+* [ ] Engineering
+* [ ] QE
+
+TBD
+
+=== Affected Projects or Components
+
+* Transactions
+* WildFly
+
+=== Other Interested Projects
+
+* MicroProfile
+* Galleon
+
+=== Relevant Installation Types
+
+* [x] Traditional standalone server (unzipped or provisioned by Galleon)
+* [ ] Managed domain
+* [ ] OpenShift s2i
+* [ ] Bootable jar
```

Comment:
we should definitely target also at least bootable jar

Reply:
And will it be easy to target the other installation types once we get some feedback on its use in traditional standalone server?

Reply:
I believe the feature pack should be reusable in the bootable jar. The bootable jar will be mostly preferred packaging for microservices-based EAP installations with XP. I don't think it would be wise to delay this work.

Reply:
I believe the point about bootable jar is valid and it would be good to add it to the scope.